src/style.css

```diff
@@ -0,0 +1,86 @@
+@font-face {
+  font-family: Avenir-Book;
+  src: url(fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir-Heavy;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: 800;
+}
+
+.bigSearch {
+  position: relative;
+  margin-top: 20px;
+  margin-bottom: 20px;
+  box-sizing: border-box;
+}
+
+.bigSearchImg {
+  position: absolute;
+  width: 19px;
+  left: 26px;
+  bottom: 26px;
+}
+
+.bigInput {
+  width: 100%;
+  height: 70px;
+  padding: 23px 0 23px 62px;
+  font-family: Avenir-Book, sans-serif;
+  font-size: 16px;
+  line-height: 22px;
+  background: #fff;
+  color: #3d4e61;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+}
+
+.bigInput:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.bigInput:focus {
+  font-family: Avenir-Heavy, sans-serif;
+  background: linear-gradient(180deg,#fff 0%,#f6f6f7 100%);
+  outline: none;
+}
+
+.smallSearch {
+  position: relative;
+}
+
+.smallInput {
+  width: 100%;
+  height: 42px;
+  padding: 11px 0 11px 33px;
+  font-family: Avenir-Book, sans-serif;
+  font-size: 14px;
+  line-height: 19px;
+  color: #3d4e61;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+}
+
+.smallSearchImg {
+  position: absolute;
+  width: 11px;
+  left: 13px;
+  bottom: 16px;
+}
+
+.smallInput:hover {
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.smallInput:focus {
+  font-family: Avenir-Heavy, sans-serif;
+  background: linear-gradient(180deg,#fff 0%,#f6f6f7 100%);
+  border-radius: 3px;
+  outline: none;
+}
```

Comment:
Here are still a lot of duplicates. These styles are duplicates of bigSearchInput, so create a class `searchInpunt` and move there all connected styles.

Check examples in [Checklist](https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md) the 2nd item

Feel free to ask me in the private channel if you don't have ideas how to fix this